.github/workflows/lint.yml

```
@@ -98,9 +98,9 @@ jobs:
           go-version-file: 'go.mod'
 
       - name: golangci-lint
-        uses: golangci/golangci-lint-action@v3
+        uses: golangci/golangci-lint-action@v6
         with:
-          version: v1.55.2
+          version: v1.58
```

```suggestion
          args: --out-format "github-actions,colored-line-number"
          version: v1.58
```

With that, we could gain useful output in just one run and we could avoid a second run.

---

Done!